.github/workflows/main.yaml

```diff
@@ -0,0 +1,30 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-deploy:
+    name: Build & Deploy
+    runs-on: [self-hosted]
+
+    steps:
+      - name: Checkout
+        run: |
+          cd ~/UML_Mentor
+          git switch main
+          git pull
+
+      - name: Build Client
+        run: |
+          cd ~/UML_Mentor/client
+          npm install && npm run build
+
+      - name: Start Server
+        run: |
+          cd ~/UML_Mentor
+          cp ~/env_file .env
+
```

Comment:
Where is the command to start the server?

Reply:
It's not necessary. PM2 is running in watch mode, which means if the files in the directory are changed the server is automatically refreshed.